staging/src/k8s.io/mount-utils/go.mod

```diff
@@ -0,0 +1,16 @@
+// This is a generated file. Do not edit directly.
+
+module k8s.io/mount-utils
+
+go 1.14
```

Comment:
This would need to be go 1.13 for the dependencies job to pass

Reply:
Update -- this isn't a bug, but intended behavior: https://github.com/kubernetes/kubernetes/pull/92611

Reply:
Thanks @nikhita for checking this up. I moved go.mod to use 'go 1.13'. Things look better